benchadapt/python/benchadapt/adapters/_adapter.py

```diff
@@ -46,6 +51,7 @@ def __init__(
         self.command = command
         self.result_fields_override = result_fields_override or {}
         self.result_fields_append = result_fields_append or {}
+        self.run_id = uuid.uuid4().hex
```

Comment:
This could just be generated in `transform_results()` and passed through to `update_result()` if we want it to differ for multiple runs with the same adapter instance

Reply:
Maybe this is a better idea

Reply:
changed